cmd/cl-controlplane/app/server.go

```diff
@@ -58,27 +58,8 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 // Run the various controlplane servers.
 func (o *Options) Run() error {
 	// set log file
-	if o.LogFile != "" {
-		f, err := os.OpenFile(o.LogFile, os.O_APPEND|os.O_CREATE|os.O_RDWR, 0666)
-		if err != nil {
-			return fmt.Errorf("unable to open log file: %v", err)
-		}
-
-		defer func() {
-			if err := f.Close(); err != nil {
-				log.Errorf("Cannot close log file: %v", err)
-			}
-		}()
```

Comment:
This cleanup is missing.
Maybe you can allow `SetLog` to accept an open file?

Reply:
Done, 77951bbaa193166b9b0d755b51fb1f4785f42eb6